core/src/main/java/org/stellar/anchor/util/ConfigHelper.java

```diff
@@ -12,30 +12,23 @@ public final class ConfigHelper {
    * provided, lookup in the collection by the domain. If it's not provided, lookup by the SEP-10
    * account.
    *
-   * @param clientsConfig collection of client configurations
+   * @param clientService client service to use for the lookup
    * @param clientDomain domain to use for the lookup
    * @param sep10Account SEP-10 authenticated account to use for lookup
    * @return client config by the predicate. If not found, returns null.
-   * @throws SepValidationException if the client config failed to validate.
    */
-  public static ClientsConfig_DEPRECATED.ClientConfig_DEPRECATED getClientConfig(
-      ClientsConfig_DEPRECATED clientsConfig, String clientDomain, String sep10Account)
-      throws SepValidationException {
-    ClientsConfig_DEPRECATED.ClientConfig_DEPRECATED clientConfig;
+  public static ClientConfig getClientConfig(
```

Comment:
Nit: Do we need this class? It doesn't seem like it does much now except wrap calls to `ClientService`.

Reply:
I'm considering merging ClientFinder and ConfigHelper into something like `SEP10ClientHelper` since both are specifically used for finding clients through SEP10